target/linux/ath79/dts/ar7100_mikrotik_routerboard-rb4xx.dtsi

```diff
@@ -0,0 +1,154 @@
+// SPDX-License-Identifier: GPL-1.0-or-later OR MIT
+
+// Based on ar71xx arch/mips/ath79/mach-rb4xx.c
+
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "ar7100.dtsi"
+
+/ {
+	chosen {
+		bootargs = "console=ttyS0,115200";
+	};
+
+	aliases {
+		led-boot = &led_user_yellow;
+		led-failsafe = &led_user_yellow;
+		led-running = &led_user_yellow;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		led_user_yellow: user_yellow {
+			label = "rb4xx:yellow:user";
+			gpios = <&gpio 4 GPIO_ACTIVE_HIGH>;
+			default-state = "on";
+		};
+
+		led_1_green: 1_green {
+			label = "rb4xx:green:led1";
+			gpios = <&cpld 0 GPIO_ACTIVE_LOW>;
+		};
+
+		led_2_green: 2_green {
+			label = "rb4xx:green:led2";
+			gpios = <&cpld 2 GPIO_ACTIVE_LOW>;
+		};
+
+		led_3_green: 3_green {
+			label = "rb4xx:green:led3";
+			gpios = <&cpld 1 GPIO_ACTIVE_LOW>;
+		};
+
+		led_4_green: 4_green {
+			label = "rb4xx:green:led4";
+			gpios = <&cpld 3 GPIO_ACTIVE_LOW>;
+		};
+
+		led_5_green: 5_green {
+			label = "rb4xx:green:led5";
+			gpios = <&cpld 8 GPIO_ACTIVE_HIGH>;
+		};
+	};
+
+	keys: keys {
+		compatible = "gpio-keys";
+
+		status = "disabled";
+
+		reset_key: reset {
```

Comment:
Do you need this label "reset_key"?

Reply:
I think so, so child dts files can have:

```
&reset_key {
  status = "okay";
}
```

Reply:
If child files are different, please set up the whole key in the child files.

Reply:
As per above, I have deleted this